Containerfile

```diff
@@ -34,4 +42,7 @@ RUN mamba install -c conda-forge nodejs \
         tweepy \
         vega_datasets
 
+# Install Pytorch via conda with CPU resources (no cuda in GCP) 
+RUN conda install pytorch torchvision torchaudio cpuonly -c pytorch
```

Comment:
Can you use mamba instead of conda here?  conda sucks.

Reply:
pytorch didn't have specific instructions for using mamba and there's changes with the CPU only since there's no cuda resources available.